src/administrator/includes/rbacl.php

```diff
@@ -199,4 +201,65 @@ public static function getRoleByUser($userId, $client = '', $clientContentIid =
 
 		return $roles;
 	}
+
+	/**
+	 * Method to Get roles of users again to selected agency.
+	 *
+	 * @param   integer  $agencyId  agency id
+	 * @param   integer  $userId    user id
+	 * @param   integer  $roleId    selected role id
+	 *
+	 * @return 	mixed
+	 *
+	 * @since 1.6
+	 */
+	public function getAuthorizedActions($agencyId = null, $userId = null, $roleId = null)
```

Comment:
Why are there hardcoded references to com_multiagency ?

Reply:
Changed the variable name.